neoutils/neoutils.go

```diff
@@ -64,9 +66,9 @@ func CheckWritable(cr CypherRunner) error {
 }
 
 // EnsureIndexes will, for a map of labels and properties, check whether an index exists for a given property on a given label, and if missing will create one
-func EnsureIndexes(im IndexManager, indexes map[string]string) error {
+func EnsureIndexes(im IndexManager, indexes map[string]string, log *logger.UPPLogger) error {
```

Comment:
We can make the logger optional here as well meaning allowing to give nil as logger object.